composer.json

```diff
@@ -0,0 +1,48 @@
+{
+    "name": "connectholland/user-bundle",
+    "description": "User bundle for Symfony 4 projects",
+    "type": "symfony-bundle",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Reyo Stallenberg",
+            "email": "reyo@connectholland.nl"
+        }
+    ],
+    "require": {
+        "php": "^7.3",
+        "symfony/form": "^3.4 || ^4.3",
+        "symfony/framework-bundle": "^3.4 || ^4.3",
+        "symfony/http-kernel": "^3.4 || ^4.3",
+        "symfony/security-bundle": "^3.4 || ^4.3",
+        "symfony/templating": "^3.4 || ^4.3",
+        "symfony/translation": "^3.4 || ^4.3",
+        "symfony/twig-bundle": "^3.4 || ^4.3",
+        "symfony/validator": "^3.4 || ^4.3",
+        "twig/twig": "^2.0 || ^3.0"
+    },
```

Comment:
Maybe you should analyse this using https://packagist.org/packages/maglnet/composer-require-checker, I see getDoctrine in the code, but I think it is missing here.

Reply:
Thanks, will do it for a following PR